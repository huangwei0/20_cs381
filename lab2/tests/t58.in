debug off
stopwords off
load tracks.txt
preprocess
length 1
sequence I
sequence have
sequence a
sequence lot
sequence to
sequence say
length 100
sequence about 