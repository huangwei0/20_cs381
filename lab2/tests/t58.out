i love
have you
a little
lot of
to the
say goodbye
about you are the world is a little girl in my heart of love me to be your eyes on fire and i don't know what it up for christmas song from here comes down by myself again kathleen intro 2 u want more than this time after all over now that way out there was made honky tonk train blues away with no one day at night long as we go home tonight i'm not alone together forever young man who do something good times roll star spangled banner waving goodbye baby blue moon river valley pta meeting place like
